scripts/signedbuildskadi.sh

```diff
@@ -44,6 +45,7 @@ else
     NGINX_PASSWORD="skadi"
     GRAFANA_USER=$NGINX_USER
     GRAFANA_PASSWORD=$NGINX_PASSWORD
+    HEAP_SIZE=$(HEAP_SIZE:-1g)
```

Comment:
After testing the script, I don't think these actually need to be here. Let's try removing both of these and then we can merge and test

Reply:
latest commit fixes this